docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Text/ApplyCustomWordEmbedding.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class ApplyCustomWordEmbedding
+    {
+        public static void Example()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var mlContext = new MLContext();
+
+            // Create an empty data sample list. The 'ApplyWordEmbedding' does not require training data as
+            // the estimator ('WordEmbeddingEstimator') created by 'ApplyWordEmbedding' API is not a trainable estimator.
+            // The empty list is only needed to pass input schema to the pipeline.
+            var emptySamples = new List<TextData>();
+
+            // Convert sample list to an empty IDataView.
+            var emptyDataView = mlContext.Data.LoadFromEnumerable(emptySamples);
+
+            var pathToCustomModel = @".\custommodel.txt";
+            using (StreamWriter file = new StreamWriter(pathToCustomModel, false))
+            {
+
+                file.WriteLine("This is custom file for 4 words with 3 dimensional word embedding vector. This first line in this file does not confirm to the '<word> <float> <float> <float>' pattern, and is therefore ignored");
+                file.WriteLine("greate" + " " + string.Join(" ", 1.0f, 2.0f, 3.0f));
```

Comment:
>"greate" + " " + string.Join(" ", 1.0f, 2.0f, 3.0f) [](start = 31, length = 51)

i think it's more readable to write this as "great 1.0 2.0 3.0"
and just add a comment above this line that the following three lines are in <word> <float> <float> <float> format.

It would also be cleaner to turn the line above to comments:
// Write a custom 3-dimensional word embedding vector with 4 words.
// The vectors follow '<word> <float> <float> <float>' pattern.